packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -360,12 +323,18 @@ export default function App() {
             theme={rainbowkitTheme}
             {...rainbowKitProviderProps}
           >
-            <WelcomeDialog {...welcomeDialogProps} onClose={onClose} />
-            <NetworkReady>
-              <AppContextProvider>
-                <Injector>{isTosAccepted && <AppContent />}</Injector>
-              </AppContextProvider>
-            </NetworkReady>
+            <Header />
+            <div className="bg-gradient-overlay flex min-h-[calc(100vh-80px)] flex-col">
```

Comment:
NIT: can we use those classes directly on `main`?

Reply:
oh nice, yea i cleaned it up